src/test/java/com/globaroman/bookshopproject/config/CustomMySqlContainer.java

```diff
@@ -0,0 +1,32 @@
+package com.globaroman.bookshopproject.config;
+
+import org.testcontainers.containers.MySQLContainer;
+
+public class CustomMySqlContainer extends MySQLContainer<CustomMySqlContainer> {
+    public static final String DB_IMAGE = "mysql:8";
+    private static CustomMySqlContainer mySqlContainer;
+
+    private CustomMySqlContainer() {
+        super(DB_IMAGE);
+    }
+
+    public static synchronized CustomMySqlContainer getInstance() {
```

Comment:
bad way of implementation the Singleton pattern
synchronized is potentially bottleneck. 
```suggestion
    public static synchronized CustomMySqlContainer getInstance() {
```

Reply:
Probably! But just used example that showed Bogdan in the video about TestContainers

Reply:
Probably! But I just used example that showed Bogdan in video about TestContainers